app/javascript/src/utils/getUserData.js

```diff
@@ -1,23 +0,0 @@
-export function getUserData() {
```

Comment:
Someone on the team can validate this, but the only references I saw to `getUserData` were in its own unit tests.

Reply:
Seems like we've been relying on `getUserDataAndCsrfToken`. Thanks for removing it!